aspnet/getting-started/installing-on-windows.rst

```diff
@@ -6,13 +6,18 @@ By `Steve Smith`_
 This article describes how to install ASP.NET 5 on Windows, showing both standalone installation as well as installation with Visual Studio 2015. 
 
 In this article:
-	- `Install ASP.NET with Visual Studio`_
-	- `Install ASP.NET Standalone`_
+  - `Install ASP.NET with Visual Studio`_
+  - `Install ASP.NET Standalone`_
 
 Install ASP.NET with Visual Studio
 ----------------------------------
 
-The easiest way to get started building application with ASP.NET 5 is to install the latest version of Visual Studio 2015 (including the freely available Community edition). Visual Studio is an Integrated Development Environment (IDE), which means it's not just an editor, but also many of the tools you need to build applications, in this case ASP.NET 5 web applications. When installing Visual Studio 2015, you'll want to be sure to specify that you want to install the Microsoft Web Developer Tools.
+The easiest way to get started building applications with ASP.NET 5 is to install the latest version of Visual Studio 2015 (including the freely available Community edition). Visual Studio is an Integrated Development Environment (IDE), which means it's not just an editor, but also has many of the tools you need to build applications, in this case ASP.NET 5 web applications. 
+
+1. Install `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=532606>`__
+2. Install the latest `ASP.NET 5 preview (Beta7) <http://go.microsoft.com/fwlink/?LinkId=623894>`__
+
+When installing Visual Studio 2015, you'll want to be sure to specify that you want to install the Microsoft Web Developer Tools.
```

Comment:
I'd like to avoid duplicating the installation instructions on the Download Center page for beta7, but I have updated the fwlink so that the installation instructions are now displayed on the Download Center page when you follow the link and I added some text to call people's attention to these instructions.